packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -165,6 +165,17 @@ export default class SelectNodeType extends PureComponent {
 
     handleNodeTypeFilterChange = filterSearchTerm => this.setState({filterSearchTerm});
 
+    componentDidUpdate() {
+        const {insertMode} = this.state;
+        if (insertMode === 'into' &&
+            this.getAllowedNodeTypesByCurrentInsertMode().length === 1 &&
+            this.getAllowedNodeTypesByCurrentInsertMode()[0].nodeTypes.length === 1) {
```

Comment:
I guess this would result in error if there's no allowed nodetype. Usually shouldn't happen, but an extra safety wouldn't hurt, at least till we have TS.

Reply:
Then `insertMode` should be blank.

Reply:
Ok, let's hope that it doesn't accidentally break, but I guess TS would force a check here.